# Introduction

#### My report is focus on a travel guide for people going to visit Toronto but not familiar with there. My work is to simplify various venues and only list the most important and attractive venues

# Business Problem

#### Nowadays, people because of the more and more busy work, have restricted time to research a place and make a travel plan before vacation. In addition, as a new visitor it is very difficult to find out best places to have fun or best restaurant to have dinner. Besides, in order to travel on schedule, the transportation and hotel are also very important. 
#### Previously, I retrieved all venues in East, West and Central Toronto as below:

In [6]:
import pandas as pd
Toronto_Dict_csv = 'Toronto Venues.csv'   
toronto_venues = pd.read_csv(Toronto_Dict_csv)    # read Toronto Postcode and geospatial data
print(toronto_venues.shape)
toronto_venues.head(10)  # check postcode data

(468, 7)


Neighbourhood  Neighbourhood Latitude  \
0                   The Beaches               43.676357   
1                   The Beaches               43.676357   
2                   The Beaches               43.676357   
3  The Danforth West, Riverdale               43.679557   
4  The Danforth West, Riverdale               43.679557   
5  The Danforth West, Riverdale               43.679557   
6  The Danforth West, Riverdale               43.679557   
7  The Danforth West, Riverdale               43.679557   
8  The Danforth West, Riverdale               43.679557   
9  The Danforth West, Riverdale               43.679557   

   Neighbourhood Longitude                    Venue  Venue Latitude  \
0               -79.293031        Glen Manor Ravine       43.676821   
1               -79.293031        Glen Stewart Park       43.675278   
2               -79.293031      Glen Stewart Ravine       43.676300   
3               -79.352188                 Pantheon       43.677621   
4               -79.352188             Dolce Gelato       43.677773   
5               -79.352188            MenEssentials       43.677820   
6               -79.352188               La Diperie       43.677530   
7               -79.352188  Messini Authentic Gyros       43.677827   
8               -79.352188     Moksha Yoga Danforth       43.677622   
9               -79.352188   Louis Cifer Brew Works       43.677663   

   Venue Longitude        Venue Category  
0       -79.293942                 Trail  
1       -79.294647                  Park  
2       -79.294784  Other Great Outdoors  
3       -79.351434      Greek Restaurant  
4       -79.351187        Ice Cream Shop  
5       -79.351265        Cosmetics Shop  
6       -79.352295        Ice Cream Shop  
7       -79.350569      Greek Restaurant  
8       -79.352116           Yoga Studio  
9       -79.351313               Brewery

In [4]:
print('There are {} uniques categories.'.format(len(toronto_venues['Venue Category'].unique())))

There are 150 uniques categories.


#### We can see there are 150 categories contain Trail, Park, Other Great Outdoors and Ice Cream Shop, et al. So many different venue categories confuse people and make it hard to make a travel plan. Thus, simplifying the venue categories is necessary for new visitors.
#### The first step is ignoring the unnecessary venue, and then classify the rest categories into 6 groups: Entertainment, Food, Hotel, Scenic spot, Shop, Transportation. More details are described in Data Section.
